fence/README.md

```diff
@@ -0,0 +1,153 @@
+# spec
```

Comment:
`spec` as title of the document?

Reply:
Looks like I pasted it from my notes XD. Will update it!